scripts/package-build/build.py

```
@@ -136,9 +136,26 @@ def build_package(package: list, patch_dir: Path) -> None:
             if (repo_dir / 'patches'):
                 apply_patches(repo_dir, patch_dir / repo_name)
 
-        # Sanitize the commit ID and build a tarball for the package
-        commit_id_sanitized = package['commit_id'].replace('/', '_')
-        tarball_name = f"{repo_name}_{commit_id_sanitized}.tar.gz"
+        # Create original tarball for dpkg-source to be happy
+        package_version = run(['dpkg-parsechangelog', '--show-field', 'Version', '--file', repo_dir / 'debian/changelog'], capture_output=True, check=False)
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)